control/veda_hotstore.py

```diff
@@ -1,13 +1,16 @@
 
-import os
-import sys
 import boto
 import boto.s3
 from boto.s3.key import Key
-import yaml
-import shutil
-from os.path import expanduser
+from boto.exception import S3ResponseError
 import newrelic.agent
+import os
+from os.path import expanduser
+import shutil
+import sys
+import yaml
```

Comment:
Pep8 prefers the library imports before the application's imports. Also, we're adding isort to sort these imports which also make sure that imports are alphabetically ordered.